nais_example.yaml

```diff
@@ -17,10 +17,10 @@ healthcheck: #Optional
     path: isready
     initialDelay: 20
     timeout: 1
-#Optional. Defaults to NONE.
-#See https://kubernetes.io/docs/concepts/containers/container-lifecycle-hooks/
 leaderElection: false # if true, a http endpoint will be available at $ELECTOR_PATH that return the current leader
                       # Compare this value with the $HOSTNAME to see if the current instance is the leader
+#Optional. Defaults to NONE.
+#See https://kubernetes.io/docs/concepts/containers/container-lifecycle-hooks/
```

Comment:
Should add example doc.  WIth a few words on what kind of cluster is created and how they can use  access the redis cluster through a k8s service.

Reply:
Should we really have that much text in the example-file? Isn't it better to have it in the doc-repo?

Reply:
I think we should have a simple example. Details in the doc.